README.md

```diff
@@ -2,6 +2,8 @@
 ## Introduction
 `rpi23-gen-image.sh` is an advanced Debian Linux bootstrapping shell script for generating Debian OS images for Raspberry Pi 2 (RPi2) and Raspberry Pi 3 (RPi3) computers. The script at this time supports the bootstrapping of the Debian (armhf) releases `jessie` and `stretch`. Raspberry Pi 3 images are currently generated for 32-bit mode only.
 
+Note: This version of the image builder is used to generate custom Sugar Debian images for the Raspberry pi 2 & 3. Please use the rpi3stretch-sugar or the rpi2stretch-sugar template to run the build process
```

Comment:
- [ ] _version_ could be _fork_ or _branch_,
- [ ] mention of _Raspberry pi 2 & 3_ is superfluous, since it is already the scope set by previous paragraph,
- [ ] you do not say how to run the build process with a template; perhaps refer to the templates section.

Reply:
Okay will make the proposed changes